src/checker.ts

```diff
@@ -211,6 +216,21 @@ export function makeFunctionlessChecker(
     );
   }
 
+  function isApiIntegrationsStaticMethod(
```

Comment:
I removed that, was thinking about promoting the static constructors and even making the classes private because you lose type checking

I can add them back in though, it's easy

Reply:
added it back